make/testing_flags.mk

```diff
@@ -57,6 +59,12 @@ ifneq ($(tags),)
 DEV_TAGS += ${tags}
 endif
 
+# Enable integration test coverage (requires Go >= 1.20.0).
+ifneq ($(cover),)
+ITEST_COVERAGE = -cover
```

Comment:
Things work a bit differently in our integration test as compared to `lnd`'s.
In `lnd` we compile a binary of the integration test (`go build -cover`), then run that binary multiple times in parallel, which generates the "mergeable" cover format.

Since we here run everything in a single invocation of `go test`, we can actually just specify `-coverprofile=coverage-itest.txt` and use that directly in `path-to-profile: coverage-itest.txt` of the coverage plugin.

So there's no need for the `COLLECT_ITEST_COVERAGE` step.

Reply:
From offline update, that change didn't work out.

I pulled the itest changes out of this PR and put them on the branch https://github.com/lightninglabs/taproot-assets/tree/coveralls_unit_itest so we can pick that up independently.